src/components/Signin.jsx

```diff
@@ -162,7 +162,7 @@ const Signin = () => {
                                     name="password"
                                     value={password}
                                     onChange={e => onChange(e)}
-                                    placeholder="Write the subject of the mail here."
+                                    placeholder="Enter your password here."
                                     required
                                 />
```

Comment:
SignIn and Sign Out buttons ?, heading as well as all the oAuth Options. Correct that

Reply:
ok I thought someone less is  working on that, will do that as well